Arduino/McLighting/request_handlers.h

```diff
@@ -277,7 +321,7 @@ void handleSetWS2812FXMode(uint8_t * mypayload) {
 String listStatusJSON(void) {
   uint8_t tmp_mode = (mode == SET_MODE) ? (uint8_t) ws2812fx_mode : strip.getMode();
   
-  const size_t bufferSize = JSON_ARRAY_SIZE(3) + JSON_OBJECT_SIZE(6);
+  const size_t bufferSize = JSON_ARRAY_SIZE(3) + JSON_OBJECT_SIZE(6) + 500;
```

Comment:
Latest Arduino JSON version is very aggressive with memory managgrment, it won't send any data just because of buffer length being small. The previous code without those changes would not work correctly.